man/IRkernel.Rd

```diff
@@ -9,9 +9,7 @@
 \title{An R kernel for Jupyter.}
 \format{An object of class \code{list} of length 5.}
 \usage{
-options(jupyter.* = ...)
-getOption('jupyter.*')
-jupyter_option_defaults$jupyter.*
+jupyter_option_defaults
```

Comment:
Not sure if there is a better way... Probably with examples, but the code must be valid R, so the `jupyter.*` syntax is out...